### ISSUE TITLE
Crash on installing Firefox Nightly for Android

### ISSUE THREAD

**Issue by porcospino44** [Tobi823/ffupdater#366]
As the title says. I had been using Firefox Nightly for a long time but it stopped updating around February 13. I uninstalled the browser and tried to reinstall it with FFUpdater but I get the following error message.

`Stacktrace:
```
java.lang.Exception: Can't find latest update for FIREFOX_NIGHTLY.
	at de.marmaro.krt.ffupdater.storage.MetadataCache.getCachedOrFetchIfOutdated(MetadataCache.kt:33)
	at de.marmaro.krt.ffupdater.storage.MetadataCache$getCachedOrFetchIfOutdated$1.invokeSuspend(MetadataCache.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8741)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@3bc9188, Dispatchers.Main]
Caused by: java.lang.Exception: Can't find latest update for FirefoxNightly.
	at de.marmaro.krt.ffupdater.app.impl.AppBase.findAppUpdateStatus(AppBase.kt:96)
	at de.marmaro.krt.ffupdater.app.impl.AppBase$findAppUpdateStatus$1.invokeSuspend(AppBase.kt:0)
	... 10 more
Caused by: java.lang.NullPointerException
	at de.marmaro.krt.ffupdater.network.mozillaci.MozillaCiJsonConsumer.updateCheck(MozillaCiJsonConsumer.kt:35)
	at de.marmaro.krt.ffupdater.network.mozillaci.MozillaCiJsonConsumer$updateCheck$1.invokeSuspend(MozillaCiJsonConsumer.kt:0)
	... 10 more
Timestamp: 2023-03-06T22:56:07.497

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.7 (133) release |
| Device | SM-A325F (a32eea, a32, a32) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | VPDJR210, dpi, release-keys, 1673322111000 |         `

**Comment by Tobi823** [comment]
Should be fixed in version 77.7.7. Can you update to FFUpdater 77.7.9 and try again?

**Comment by porcospino44** [comment]
Yep, it works. Thank you.
